sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/internal/utils/DatabaseUtils.java

```diff
@@ -0,0 +1,151 @@
+                    getServiceReference(DataSourceService.class);
+            if (serviceRef == null) {
+                throw new GeoVelocityException("Cannot find the datasourceService '" +
+                        DataSourceService.class.getName() + "'");
+            } else {
+                DataSourceService dataSourceService = bundleContext.getService(serviceRef);
+                dataSource = (HikariDataSource) dataSourceService.getDataSource(dataSourceName);
+                if (log.isDebugEnabled()) {
+                    log.debug("Lookup for datasource '" + dataSourceName + "' completed through " +
+                            "DataSource Service lookup.");
+                }
+            }
+        } catch (DataSourceException e) {
+            throw new GeoVelocityException("Cannot connect to the datasource '" + dataSourceName + "'", e);
+        }
+    }
+
+    /**
+     * Utility method to get a new database connection.
+     *
+     * @return Connection
+     * @throws SQLException if failed to get Connection
+     */
+    public Connection getConnection() throws SQLException {
+
+        if (dataSource != null) {
+            return dataSource.getConnection();
+        }
+        throw new SQLException("Data source is not configured properly.");
+    }
+
+    /**
+     * Utility method to close the connection streams.
+     *
+     * @param preparedStatement PreparedStatement
+     * @param connection        Connection
+     * @param resultSet         ResultSet
+     */
+    public void closeAllConnections(PreparedStatement preparedStatement, Connection connection,
+                                    ResultSet resultSet) {
+
+        closeResultSet(resultSet);
+        closeStatement(preparedStatement);
+        closeConnection(connection);
+    }
+
+    /**
+     * Close Connection.
+     *
+     * @param dbConnection Connection
+     */
+    private void closeConnection(Connection dbConnection) {
+
+        if (dbConnection != null) {
+            try {
+                dbConnection.close();
+            } catch (SQLException e) {
+                log.error("Couldn't close connection", e);
+            }
+        }
+    }
+
+    /**
+     * Close ResultSet.
+     *
+     * @param resultSet ResultSet
+     */
+    private void closeResultSet(ResultSet resultSet) {
+
+        if (resultSet != null) {
+            try {
+                resultSet.close();
+            } catch (SQLException e) {
+                log.error("Couldn't close ResultSet", e);
+            }
+        }
+
+    }
+
+    /**
+     * Close PreparedStatement.
+     *
+     * @param preparedStatement PreparedStatement
+     */
+    private void closeStatement(Statement preparedStatement) {
+        if (preparedStatement != null) {
+            try {
+                preparedStatement.close();
+            } catch (SQLException e) {
+                log.error("Couldn't close Statement", e);
+            }
+        }
+
```

Comment:
unnecessary new line

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265